altair/vegalite/v3/tests/test_api.py

```diff
@@ -858,4 +858,9 @@ def test_graticule():
     assert data.to_dict() == {'graticule': True}
 
     data = alt.graticule(step=[15, 15])
-    assert data.to_dict() == {'graticule': {'step': [15, 15]}}
\ No newline at end of file
+    assert data.to_dict() == {'graticule': {'step': [15, 15]}}
+
+
+def test_sphere():
+    data = alt.sphere()
+    assert data.to_dict() == {'sphere': True}
```

Comment:
Test passing of kwargs as well - anything is allowed according to the schema, and ``alt.sphere(key='val').to_dict()`` should result in ``{'sphere': {'key': 'val'}}``